docs/source/llms/transformers/tutorials/fine-tuning/transformers-peft.ipynb

```
@@ -0,0 +1,1281 @@
+      "| NVIDIA-SMI 535.54.03              Driver Version: 535.54.03    CUDA Version: 12.2     |\r\n",
+      "|-----------------------------------------+----------------------+----------------------+\r\n",
+      "| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |\r\n",
+      "| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |\r\n",
+      "|                                         |                      |               MIG M. |\r\n",
+      "|=========================================+======================+======================|\r\n",
+      "|   0  NVIDIA A10G                    Off | 00000000:00:1E.0 Off |                    0 |\r\n",
+      "|  0%   15C    P8              16W / 300W |      4MiB / 23028MiB |      0%      Default |\r\n",
+      "|                                         |                      |                  N/A |\r\n",
+      "+-----------------------------------------+----------------------+----------------------+\r\n",
+      "                                                                                         \r\n",
+      "+---------------------------------------------------------------------------------------+\r\n",
+      "| Processes:                                                                            |\r\n",
+      "|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |\r\n",
+      "|        ID   ID                                                             Usage      |\r\n",
+      "|=======================================================================================|\r\n",
+      "|  No running processes found                                                           |\r\n",
+      "+---------------------------------------------------------------------------------------+\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "% nvidia-smi"
```

Does Jupyter support this magic command?

---

What happens if I run-all this notebook and `nvidia-smi` is missing?

---

> Does Jupyter support this magic command?

Nope, fixed to `%sh nvidia-smi`.

> What happens if I run-all this notebook and nvidia-smi is not installed?

The first cell should fail with `command not found`, which is desired as we don't expect users to run this notebook without CUDA GPU backend.